mxfusion/inference/inference.py

```diff
@@ -55,6 +55,28 @@ def __init__(self, inference_algorithm, constants=None,
                                           context=self.mxnet_context)
         self._initialized = False
 
+    def pretty_print_params(self):
```

Comment:
Maybe rename to print_params.
```suggestion
    def print_params(self):
```